samples/dynamic-headers/custom-visual/capabilities.json

```diff
@@ -0,0 +1,44 @@
+{
+  "dataRoles": [
+    {
+      "displayName": "Static title columns",
```

Comment:
Consider renaming the data role to "Static column values". Propagate this change accordingly through all the implementation for consistency.